### PR TITLE
spice-gtk 0.39

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

Previous efforts: #52379, #57135, #70124

### CODE REVIEW

File: Formula/spice-gtk.rb [hunk 1]
```diff
@@ -50,29 +51,35 @@ class SpiceGtk < Formula
...
   depends_on "usbredir"
 
-  # Upstream patch: https://gitlab.freedesktop.org/spice/spice-gtk/issues/88
-  patch do
-    url "https://gitlab.freedesktop.org/spice/spice-gtk/commit/3c9b37bfc7c88969dfe16b8bfd874745e0fceb8a.diff"
-    sha256 "c2bb9c6dc0d07f333d10077987386680818296f1deb3b796ea7e35453aba7d91"
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
```
Comment (carlocab) [inline Formula/spice-gtk.rb:57]: We can replace this with `depends_on "six"`.

Comment (EricFromCanada) [reply Formula/spice-gtk.rb:58]: Normally I would, but I left it as a resource since `pyparsing` is required also.

**Review by tschoonj — APPROVED** [review]
LGTM

**Comment by AntonioMeireles** [comment]
folks, FWIW this seems broken; getting ... 
```shell
>> brew upgrade spice-gtk   
==> Upgrading 1 outdated package:
spice-gtk 0.37_4 -> 0.39
==> Upgrading spice-gtk
  0.37_4 -> 0.39

==> Downloading https://ghcr.io/v2/homebrew/core/spice-gtk/manifests/0.39
##O#- #
curl: (22) The requested URL returned error: 404
Error: Failed to download resource "spice-gtk_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/spice-gtk/manifests/0.39

```

All the best.

**Comment by carlocab** [comment]
Bottle commit pushed in https://github.com/Homebrew/homebrew-core/commit/20a2cbad928e9209466401ee491903265aa8fff6. Apologies for the inconvenience!

**Pull request merged**